pkg/graphviz/traceflow_test.go

```diff
@@ -369,3 +600,11 @@ func TestGetTraceflowStatusMessage(t *testing.T) {
 		})
 	}
 }
+
+func TestFindClusterString(t *testing.T) {
+	graphStr := ""
+	clusterName := "cluster_source"
+	_, gotEndIndex := findClusterString(graphStr, clusterName)
```

Comment:
maybe graphStr isn't required. You can remove graphStr var in Line 605
```suggestion
	_, gotEndIndex := findClusterString("", clusterName)
```